Python/di_sensors/inertial_measurement_unit.py

```diff
@@ -13,60 +13,107 @@
 
 
 class InertialMeasurementUnit(object):
+    """
+    Class for interfacing with the `InertialMeasurementUnit Sensor`_.
+    """
     def __init__(self, bus = "RPI_1"):
-        """Initialize the sensor
-        
-        Keyword arguments:
-        bus (default "RPI_1") -- The I2C bus"""
+        """
+        Constructor for initializing link with the `InertialMeasurementUnit Sensor`_.
+
+        :param str bus = "RPI_1": The bus to which the distance sensor is connected to. By default, it's set to bus ``"RPI_1"``. Check the :ref:`hardware specs <hardware-interface-section>` for more information about the ports.
+        :raises RuntimeError: When the chip ID is incorrect. This happens when we have a device pointing to the same address, but it's not a `InertialMeasurementUnit Sensor`_.
+        :raises ~exceptions.OSError: When the `InertialMeasurementUnit Sensor`_ is not reachable.
```

Comment:
how is OSError raised? I'm not seeing it in the code. Must be getting late for me.

Reply:
Take a look here: https://github.com/DexterInd/DI_Sensors/blob/master/Python/di_sensors/dexter_i2c.py#L34

This module is critical to the communications between any sensor and the Pi. When there's something misbehaving on the I2C line (sensor disconnected, high impedance, noise, etc), an `OSError` exception is thrown every time.